src/app/components/country-select/country-select.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { RegionSelectService } from 'src/app/services/region-select.service';
+
+@Component({
+  selector: 'app-country-select',
+  templateUrl: './country-select.component.html',
+  styleUrls: ['./country-select.component.scss']
+})
+export class CountrySelectComponent implements OnInit {
+  countries = ['global', 'GB'];
+  selectedCountry: string = this.countries[0];
```

Comment:
you could do away with selected country and just set the value to the first element in the html ? Its something along the lines of [value]="countries[0]"